controllers/controller.js

```diff
@@ -1,10 +1,15 @@
-const { readTopics } = require("../models/model")
+const { readTopics, readArticles } = require("../models/model");
 
 const getTopics = (req, res) => {
-    return readTopics().then((topics) => {
-        res.status(200).send(topics)
-    })
-}
+  return readTopics().then((topics) => {
+    res.status(200).send(topics);
+  });
+};
 
+const getArticles = (req, res) => {
+  return readArticles().then((articles) => {
+    res.status(200).send(articles);
+  });
```

Comment:
it's always best to have a catch block , include the `next` argument in your callback and attached a catch to your promise chain `.catch(next)` this ensure that any errors are being passed on to the app